include/wx/dynlib.h

```diff
@@ -251,10 +251,13 @@ class WXDLLIMPEXP_BASE wxDynamicLibrary
     wxDllType Detach() { wxDllType h = m_handle; m_handle = NULL; return h; }
 
     // unload the given library handle (presumably returned by Detach() before)
-    static void Unload(wxDllType handle);
+    static bool Unload(wxDllType handle, wxString* errorDest = NULL);
```

Comment:
I think we should have either `errorDest` output parameters (which would be normally called `outError` in wx API) _or_ `GetErrorStr()`, but not both.